### DISCUSSION TITLE
open /.schema/config.schema.json: no such file or directory

### DISCUSSION THREAD

**Discussion by israaaa270** [ory/hydra#2321] [Q&A] [answered]
when I run container hydra with command 
 
`docker run -d --name my-hydra -p 4444:4444 -e "DATABASE_URL=memory" oryd/hydra`

I got this in docker logs 

`time=2021-01-25T01:17:00Z level=fatal msg=Unable to instantiate configuration. audience=application error=map[message:open /.schema/config.schema.json: no such file or directory] service_name=ORY Hydra service_version=master`

but when open hydra folder this  .schema/config.schema.json  is already exists.

**Comment by aeneasr** [comment, marked as answer]
Please use `make docker` when building your own docker images.

**Reply by israaaa270** [reply]
@aeneasr  thank you
